documentation/book/con-certificate-renewal.adoc

```diff
@@ -0,0 +1,73 @@
+Manually-installed CA certificates should have their own validity period defined.
+
+When a CA certificate expires, the certificates that it has signed will fail validation, even if they were previously valid. 
+This means that, when replacing a CA certificate, you must also replace all other certificates signed by it. 
+When the replacement of a CA certificate is in progress, it is necessary for peers to trust certificates signed by either the old or the new CA. 
+This ensures the continued operation of the cluster.
+
+To allow the renewal of CA certificates without a loss of service, the Cluster Operator will initiate certificate renewal before the old CA certificates expire. 
+You can configure the renewal period in `Kafka.spec.clusterCa.renewalDays` and `Kafka.spec.clientsCa.renewalDays` (both default to 30 days). 
+The renewal period is measured backwards, from the expiry date of the current certificate.
+
+[source]
+----
+Not Before                                     Not After
+    |                                              |
+    |<--------------- validityDays --------------->|
+                              <--- renewalDays --->|
+----
+
+The behavior of the Cluster Operator during the renewal period depends on whether the relevant setting is enabled, in either `Kafka.spec.clusterCa.generateCertificateAuthority` or `Kafka.spec.clientsCa.generateCertificateAuthority`.
+
+
+== Renewal process with generated CAs
+
+The Cluster Operator performs the following process to renew CA certificates:
+
+. Generate a new CA key and certificate. The new private key replaces the old private key in the corresponding `Secret`. The new certificate is given the name `ca.crt` within the corresponding `Secret` and the old certificate is renamed `ca-_<expiry-date>_.crt`.
+
+. Restart Zookeeper nodes so that they will trust the new CA certificate.
+
+. Restart Kafka brokers so that they will trust the new CA certificate.
+
+. Restart the Topic and User Operators so that they will trust the new CA certificate.
+
+. Generate new client certificates (for Zookeeper nodes, Kafka brokers, and the entity operator) signed by the new CA.
+
+. Perform the same restarts so that clients are using certificates signed by the new CA certificate.
+
+At the end of the renewal period the Cluster Operator will remove the now expired CA certificates (those named `ca-_<expiry-date>_.crt`) from the corresponding `Secret` and perform a further round of restarts.
+
+== Renewal process with your own CA certificates
+
+At the start of the renewal period the Cluster Operator will start logging at the `WARN` level that new CA certificates and keys are needed.
+Once you have provided the new certificates and keys, the Cluster Operator performs a further set of restarts within the Kafka cluster for which the warning was issued.
+
+== Client applications
+
+The Cluster Operator is not aware of all the client applications using the Kafka cluster.
```

Comment:
Just double-checking that "not aware" is correct here.

Reply:
That is correct. I managed to miss out the negative previously.